pandas/io/formats/style.py

```diff
@@ -233,10 +233,16 @@ def __init__(
 
     def _repr_html_(self) -> str:
         """
-        Hooks into Jupyter notebook rich display system.
+        Hooks into Jupyter notebook rich display system, which calls _repr_html_ by
+        default if an object is returned at the end of a cell.
         """
+        if get_option("styler.render.repr") == "latex":
```

Comment:
do we actually still need this? or is this something controlled by the notebook itself?

Reply:
I tried to look this up, and I may be wrong but I think Notebook uses `_latex_repr_` when converting the notebook to other formats, so it is useful.

When `_repr_html_` and `_repr_latex` are present the notebook still renders the html version. And I can't find docs on native iPython how to configure it to use the other. So the `pandas` options is userfriendly override (per issue)